dashboard/app/views/home/index.html.haml

```diff
@@ -25,6 +25,10 @@
   - if current_user
     #user_hero
       = render template: 'api/user_hero'
+    - [current_user.working_on_user_scripts, current_user.completed_user_scripts].compact.reduce([], :|).each_slice(2) do |course_pair|
```

Comment:
I was thinking bitwise OR to combine the two into one list, but I think compact does what I want already. It won't include the list if it's empty.